### PR TITLE
Completely updated de_de.lang

### PR DESCRIPTION
**Pull request by Legondas**
I think I got everything translated.
If something is wrong, just notify me.

Legondas

### CODE REVIEW

File: de_de.lang [hunk 1]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.name=Himmlische Notizen
+item.celestial_notes.sun.text=Solar
+item.celestial_notes.stars_1.text=Stellar, nördlicher Quadrant
+item.celestial_notes.stars_2.text=Stellar, südlicher Quadrant
+item.celestial_notes.stars_3.text=Stellar, westlicher Quadrant
+item.celestial_notes.stars_4.text=Stellar, östlicher Quadrant
+item.celestial_notes.moon_1.text=Mond, voll
+item.celestial_notes.moon_2.text=Mond, Abnehmend drei Viertel
+item.celestial_notes.moon_3.text=Mond, Anbehmender Halbmond
+item.celestial_notes.moon_4.text=Mond, Abnehmend ein Viertel
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_4.text=Abnehmender Sichelmond
```

File: de_de.lang [hunk 2]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.moon_5.text=Mond, Neumond
+item.celestial_notes.moon_6.text=Mond, Zunehmend ein Viertel
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_6.text=Zunehmender Sichelmond
```

File: de_de.lang [hunk 3]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
 item.primordial_pearl.pearl.name=Primordialperle
 item.pech_wand.name=Pech Zauberstab
+item.celestial_notes.name=Himmlische Notizen
+item.celestial_notes.sun.text=Solar
+item.celestial_notes.stars_1.text=Stellar, nördlicher Quadrant
+item.celestial_notes.stars_2.text=Stellar, südlicher Quadrant
+item.celestial_notes.stars_3.text=Stellar, westlicher Quadrant
+item.celestial_notes.stars_4.text=Stellar, östlicher Quadrant
+item.celestial_notes.moon_1.text=Mond, voll
+item.celestial_notes.moon_2.text=Mond, Abnehmend drei Viertel
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_2.text=Abnehmender Dreiviertelmond
```

File: de_de.lang [hunk 4]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
 item.primordial_pearl.nodule.name=Primordialknolle
 item.primordial_pearl.pearl.name=Primordialperle
 item.pech_wand.name=Pech Zauberstab
+item.celestial_notes.name=Himmlische Notizen
+item.celestial_notes.sun.text=Solar
+item.celestial_notes.stars_1.text=Stellar, nördlicher Quadrant
+item.celestial_notes.stars_2.text=Stellar, südlicher Quadrant
+item.celestial_notes.stars_3.text=Stellar, westlicher Quadrant
+item.celestial_notes.stars_4.text=Stellar, östlicher Quadrant
+item.celestial_notes.moon_1.text=Mond, voll
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_1.text=Vollmond
```

File: de_de.lang [hunk 5]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.moon_2.text=Mond, Abnehmend drei Viertel
+item.celestial_notes.moon_3.text=Mond, Anbehmender Halbmond
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_3.text=Abnehmender Halbmond
```

File: de_de.lang [hunk 6]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.moon_4.text=Mond, Abnehmend ein Viertel
+item.celestial_notes.moon_5.text=Mond, Neumond
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_5.text=Neumond
```

File: de_de.lang [hunk 7]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.moon_6.text=Mond, Zunehmend ein Viertel
+item.celestial_notes.moon_7.text=Mond, zunehmender Halbmond
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_7.text=Zunehmender Halbmond
```

File: de_de.lang [hunk 8]
```diff
@@ -22,6 +22,20 @@ item.primordial_pearl.mote.name=Primordialstaub
...
+item.celestial_notes.moon_8.text=Mond, Zunehmend drei Viertel
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.celestial_notes.moon_8.text=Zunehmender Dreiviertelmond
```

File: de_de.lang [hunk 9]
```diff
@@ -42,14 +56,14 @@ item.nugget.void.name=Leermetallnugget
...
 item.nugget.rareearth.name=Seltene Erden
-item.cluster.iron.name=Nativer Eisenklumpen
-item.cluster.copper.name=Nativer Kupferklumpen
-item.cluster.tin.name=Nativer Zinnklumpen
-item.cluster.silver.name=Nativer Silberklumpen
-item.cluster.lead.name=Nativer Bleiklumpen
-item.cluster.cinnabar.name=Nativer Zinnoberklumpen
-item.cluster.gold.name=Nativer Goldklumpen
-item.cluster.quartz.name=Nativer Quarzklumpen
+item.cluster.iron.name=Nativer Eisencluster
```
Comment (Vexatos) [inline de_de.lang]: I would honestly still translate them to "Klumpen". Either that, or fix "Nativer" to "Natives" as it is a neutral noun.

File: de_de.lang [hunk 10]
```diff
@@ -151,6 +168,9 @@ item.hand_mirror.name=Magischer Handspiegel
 item.thaumometer.name=Thaumometer
 item.sanity_checker.name=Verstandsprüfer (?)
 item.resonator.name=Essentiaresonator
+item.grapple_gun.name=Arkaner Greifer
+item.grapple_gun_tip.name=Greifer-Kopf
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.grapple_gun_tip.name=Greiferkopf
```

File: de_de.lang [hunk 11]
```diff
@@ -165,19 +185,21 @@ item.amulet_vis.text=Lädt gehaltene Stäbe auf
...
-item.verdant_charm.name=Talisman eines weiten Herzens
+item.verdant_charm.name=Talisman des Herzens
 item.verdant_charm.life.text=Lebensspender
 item.verdant_charm.sustain.text=Erhalter
 item.voidseer_charm.name=Perle des Tiefblickenden
 item.voidseer_charm.text=Du schaust in pechschwarzes Nichts... Du könntest schwören dass etwas zurückstarrt...
-item.curiosity_band.name=Kopfband der Kuriositäten
-item.nimbus_ring.name=Ring des Scheins
+item.curiosity_band.name=Kopfband der Neugier
+item.cloud_ring.name=Wolkenläufer Ring
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.cloud_ring.name=Wolkenläuferring
```

File: de_de.lang [hunk 12]
```diff
@@ -165,19 +185,21 @@ item.amulet_vis.text=Lädt gehaltene Stäbe auf
...
+item.charm_undying.name=Talisman der Unsterblichkeit
 #
 # Casters & Foci
 #
 item.caster_basic.name=Zaubererhandschuh
-item.focus.name=Zauberfokus
-item.fous_blank.name=Leerer Zauberfokus
+item.focus_1.name=Leerer geringerer Fokus
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.focus_1.name=Leerer schwacher Fokus
```

File: de_de.lang [hunk 13]
```diff
@@ -165,19 +185,21 @@ item.amulet_vis.text=Lädt gehaltene Stäbe auf
...
+item.focus_2.name=Leerer verbesserter Fokus
+item.focus_3.name=Leerer größerer Fokus
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.focus_3.name=Leerer starker Fokus
```

File: de_de.lang [hunk 14]
```diff
@@ -188,6 +210,8 @@ item.creative_flux_sponge.name=Fluxschwamm
 item.turret.basic.name=Automatische Armbrust
 item.turret.advanced.name=Verbesserte Automatische Armbrust
 item.turret.bore.name=Arkaner Bohrer
+item.enchanted_placeholder.name=Jeder verzauberte Gegenstand
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.enchanted_placeholder.name=Jeglicher verzauberte Gegenstand
```

File: de_de.lang [hunk 15]
```diff
@@ -209,6 +233,8 @@ item.seal.lumber.name=Kontrollsiegel: Holzfällen
 item.seal.breaker.name=Kontrollsiegel: Blöcke Abbauen
 item.seal.use.name=Kontrollsiegel: Verwenden
 item.seal.provider.name=Kontrollsiegel: Anbieten
+item.seal.stock.name=Kontrollsiegel: Lager
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.seal.stock.name=Kontrollsiegel: Lagern
```

File: de_de.lang [hunk 16]
```diff
@@ -209,6 +233,8 @@ item.seal.lumber.name=Kontrollsiegel: Holzfällen
...
+item.seal.breaker_advanced.name=Kontrollsiegel: Verbesserter Block Brecher
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.seal.breaker_advanced.name=Kontrollsiegel: Verbessertes Blöcke-Abbauen
```

File: de_de.lang [hunk 17]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
 tile.essentia_input.name=Essentiatransfusion
 tile.essentia_output.name=Essentiakatheter
 tile.activator_rail.name=Arcane Aktivierungsschiene
+tile.potion_sprayer.name=Tranksprüher
+tile.vis_generator.name=Vis Generator
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.vis_generator.name=Vis-Generator
```

File: de_de.lang [hunk 18]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
+tile.stabilizer.name=Stabilisator
+tile.condenser.name=Fluxkondensator
+tile.condenser_lattice.name=Fluxkondensatorgitter
+tile.condenser_lattice_dirty.name=Verstopftes Fluxkondensatorgitter
+tile.void_siphon.name=Leeren Siphon
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.void_siphon.name=Leersiphon
```
I translated all references to the void like this, I think it sounds pretty good.

File: de_de.lang [hunk 19]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
+tile.infernal_furnace.name=Höllenofen
+tile.inlay.name=Redstone Inlay
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.inlay.name=Redstone-Intarsie
```

File: de_de.lang [hunk 20]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
+tile.candle_purple.name=Violette Talgkerze
+tile.candle_blue.name=Blaue Talgkerze
+tile.candle_brown.name=Braune Talgkerze
+tile.candle_green.name=Grüne Talgkerze
+tile.candle_red.name=Rote Talgkerze
+tile.candle_black.name=Schwarze Talgkerze
+tile.pedestal_arcane.name=Arkaner Sockel
+tile.pedestal_eldritch.name=Unnatürlicher Sockel
+tile.pedestal_ancient.name=Antiker Sockel
+tile.banner_crimson_cult.name=Karmesinroter Kult Banner
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.banner_crimson_cult.name=Karmesinkultbanner
```

File: de_de.lang [hunk 21]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
+tile.banner_white.name=Weises Banner
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.banner_white.name=Weißes Banner
```

File: de_de.lang [hunk 22]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
+tile.banner_orange.name=Orangenes Banner
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.banner_orange.name=Oranges Banner
```

File: de_de.lang [hunk 23]
```diff
@@ -320,75 +347,101 @@ tile.brain_box.name=Mnemonische Matrix
...
-tile.paving_stone.travel.name=Pflasterstein des Reisens
-tile.paving_stone.travel_lit.name=Schimmernder Pflasterstein des Reisens
+tile.flesh.block.name=Fleischblock
+tile.paving_stone_barrier.name=Bannstein
+tile.paving_stone_travel.name=Pflasterstein des Reisens
 tile.centrifuge.name=Essentia Zentrifuge
 tile.mirror.name=Magischer Spiegel
 tile.mirror_essentia.name=Essentia Spiegel
 tile.golem_builder.name=Golempresse
+tile.vis_battery.name=Vis Batterie
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
tile.vis_battery.name=Vis-Batterie
```

File: de_de.lang [hunk 24]
```diff
@@ -400,20 +453,21 @@ enchantment.special.sapgreat=Starkes Zehren
...
 ###################################
-potion.fluxtaint=Fluxvergiftung
+potion.flux_taint=Fluxvergiftung
 potion.soulshatter=Seelensplittern
-potion.visexhaust=Fluxgrippe
+potion.vis_exhaust=Fluxgrippe
 potion.infvisexhaust=Fluxphage
 potion.unhunger=Unnatürlicher Hunger
-potion.warpward=Warpbann
+potion.warpward=Verzerrungsbann
```
Comment (Vexatos) [inline de_de.lang]: "Warp" is not being translated. It just doesn't work. Refer to https://github.com/Azanor/thaumcraft-5/blob/master/de_DE.lang and its history for this.
```suggestion
potion.warpward=Warpbann
```

File: de_de.lang [hunk 25]
```diff
@@ -531,58 +579,33 @@ tile.eldritch.lock.name=Antiker Verriegelungsmechanismus
...
-
-item.creative_placer.caster.name=Platzierer für Antike Automagier
-
-
 #
 # Text
+research.complete=Forschung abgeschlossen!
 item.runic.charge=Runenschutz
-item.warping=Warping
+item.warping=Verzerrung
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
item.warping=Warping
```
As mentioned before, this word cannot be adequately translated.

File: de_de.lang [hunk 26]
```diff
@@ -531,58 +579,33 @@ tile.eldritch.lock.name=Antiker Verriegelungsmechanismus
...
 item.capacity.text=Kapazität
 item.chargebonus.text=Ladung
+stability.gain=gewinn / Zyklus
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
stability.gain=%s Gewinn / Zyklus
```

File: de_de.lang [hunk 27]
```diff
@@ -531,58 +579,33 @@ tile.eldritch.lock.name=Antiker Verriegelungsmechanismus
...
+stability.range=0 bis
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
```
This line doesn't exist anymore?

File: de_de.lang [hunk 28]
```diff
@@ -531,58 +579,33 @@ tile.eldritch.lock.name=Antiker Verriegelungsmechanismus
...
 # Text
+research.complete=Forschung abgeschlossen!
 item.runic.charge=Runenschutz
-item.warping=Warping
+item.warping=Verzerrung
 item.capacity.text=Kapazität
 item.chargebonus.text=Ladung
+stability.gain=gewinn / Zyklus
+stability.range=0 bis 
+stability.loss=verlust / Zyklus
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
stability.loss=0 bis %s Verlust / Zyklus
```

File: de_de.lang [hunk 29]
```diff
@@ -805,6 +850,10 @@ golem.prop.redon=Beachtete Redstonesignale
 golem.prop.redoff=Ignoriert Redstonesignale
 golem.prop.exist=Behälter muss den Gegenstand bereits enthalten
 golem.prop.leave=Behalte immer einen Gegenstand
+golem.prop.silk=Benutze Behutsamkeit
+
+focuspart.com=Complexity:
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
focuspart.com=Komplexität:
```

File: de_de.lang [hunk 30]
```diff
@@ -683,67 +706,87 @@ golem.stay=Ich werde hier bleiben, Meister
...
+thaumcraft.SCATTER.name=Streuung
+thaumcraft.SCATTER.text=Du teilst die verschossenen Projektile. Diese sind schwächer und weniger akkurat, allerings haben deine Projektile eine größere Angriffsfläche (AOE).
+thaumcraft.SPLITTARGET.name=Ziel teilen
+thaumcraft.SPLITTARGET.text=Teilen Sie ein einzelnes Ziel in zwei schwächere auf.
+thaumcraft.SPLITTRAJECTORY.name=Geteilte Flugbahn
+thaumcraft.SPLITTRAJECTORY.text=Teilen Sie eine einzelne Flugbahn in zwei schwächere auf.
+#
+# SETTING DESCRIPTIONS
+focus.common.power=Macht
+focus.common.duration=Dauer (Sekunen)
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
focus.common.duration=Dauer (Sekunden)
```

File: de_de.lang [hunk 31]
```diff
@@ -683,67 +706,87 @@ golem.stay=Ich werde hier bleiben, Meister
...
+focus.common.no=Nein
+focus.common.yes=Ja
+focus.common.fortune=Glück
+focus.common.target=Zieltyp
+focus.common.friend=Freundlich
+focus.common.enemy=Feindlich
+focus.common.none=Keiner
+focus.common.options=Optionen
+focus.fire.burn=Branddauer (Sekunden)
+focus.scatter.cone=Streuung (grad)
```
Comment (Vexatos) [inline de_de.lang]:
```suggestion
focus.scatter.cone=Streuung (Grad)
```

File: de_de.lang [hunk 32]
```diff
@@ -683,67 +706,87 @@ golem.stay=Ich werde hier bleiben, Meister
...
+thaumcraft.FIRE.text=Schickt Flammen deinem Ziel entgegen und setzt es in Brand. 
+thaumcraft.FROST.name=Frost
+thaumcraft.FROST.text=Wirf deinem Ziel eisige Kälte entgegen um es zu verletzen und einzufrieren. Gefriert Wasser und verlangsamt Kreaturen.
+thaumcraft.BREAK.name=Zerstörung
+thaumcraft.BREAK.text=Beschwört eine zerstörerische Energie die die meisten Materialien bricht.
+thaumcraft.CURSE.name=Fluch
+thaumcraft.CURSE.text=Beschwört die Macht der Entropie um gezielt Kreaturen zu verletzen.
+thaumcraft.FLUX.name=Flux
+thaumcraft.FLUX.text=Dieser Effekt beschwört rohes, unfokussiertes Vis, das lebende (und tote) Kreaturen unterbricht. Diese Energie kann nicht mit unbelebten Objekten interagieren, umgeht jedoch gewöhnliche Rüstungen.
+thaumcraft.RIFT.name=Raumloch
```
Comment (Vexatos) [inline de_de.lang:728]: I like this word.

**Review by Vexatos — CHANGES_REQUESTED** [review]
All in all, this translation is... decent, the item names are pretty good, but it also uses formal language where it definitely shouldn't, and a lot of the humour in the Thaumonomicon seems to have gone missing. I will go through more of the book later. Please give me some time to fully review it as this also takes a lot of effort.

In any case, I appreciate that you spent the time working on this, even though you still need to work on using the language properly (most importantly hyphen and comma placement). With the item names, you mostly did well adopting the style of the translation.

**Comment by Vexatos** [comment]
This also opens up the good old can of worms again of whether or not to translate "taint", since in this version it's sometimes translated and sometimes not.

File: de_de.lang (hunk 9 above)
Comment (Legondas) [reply de_de.lang]: Yea changed that

**Comment by Legondas** [comment]
Thanks for your review. I think I need a lot more practice.

**Comment by Legondas** [comment]
> This also opens up the good old can of worms again of whether or not to translate "taint", since in this version it's sometimes translated and sometimes not.

I was unsure. I think in the previous translation it was sometimes changed.

**Comment by Vexatos** [comment]
Exactly, and let's not touch that for now because it is a problem for another day, let's leave it as it is.

**Pull request closed without merging**